Update README.md (#10)
polkadot-js/phishing: README.md

A curated list of known less-than-honest sites inclusive of a simple JS utility function to check any host against this list.

Any additions can be made by editing [all.json](https://github.com/polkadot-js/phishing/edit/master/all.json) and adding any new sites in alphabetical order.

## Contributing

This list is maintained by the community, and is currently available at https://github.com/polkadot-js/phishing

> Contributions to the list is welcome, either via a pull request or by [logging an issue](https://github.com/polkadot-js/phishing/issues).

Author: ```suggestion
Contributions are welcome, either via a pull request or by [logging an issue](https://github.com/polkadot-js/phishing/issues).
```
